src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -209,6 +211,34 @@ def subvolume_getpath(self, **kwargs):
             ret = self.volume_exception_to_retval(ve)
         return ret
 
+    def subvolume_info(self, **kwargs):
+        ret        = None
+        volname    = kwargs['vol_name']
+        subvolname = kwargs['sub_name']
+        groupname  = kwargs['group_name']
+
+        try:
+            with open_volume(self, volname) as fs_handle:
+                with open_group(fs_handle, self.volspec, groupname) as group:
+                    with open_subvol(fs_handle, self.volspec, group, subvolname) as subvolume:
+                        subvolpath = subvolume.path
+                        st = subvolume.fs.statx(subvolpath, cephfs.CEPH_STATX_BTIME |
+                                                            cephfs.CEPH_STATX_SIZE,
+                                                            cephfs.AT_SYMLINK_NOFOLLOW)
+                        usedbytes = st["size"]
+                        try:
+                            nsize = int(subvolume.fs.getxattr(subvolpath, 'ceph.quota.max_bytes').decode('utf-8'))
+                        except cephfs.NoData:
+                            nsize = 0
+                        data_pool = subvolume.fs.getxattr(subvolpath, 'ceph.dir.layout.pool').decode('utf-8')
+                        ret = 0, json.dumps({'subvolpath': subvolpath.decode("utf-8"), 'bytes_quota': "no-limit" if nsize == 0 else nsize,
+                            'bytes_pcent': "no-limit" if nsize == 0 else '{0:.2f}'.format((float(usedbytes) / nsize) * 100.0),
+                            'data_pool': data_pool, 'createdAt': str(st["btime"])},
```

Comment:
naming of keys could be uniform -- `created_at` just like `bytes_pcent`?

Reply:
Done